lib/hammer_cli_foreman_virt_who_configure/system_caller.rb

```diff
@@ -1,15 +1,22 @@
 module HammerCLIForemanVirtWhoConfigure
   class SystemCaller
+    WHITELISTED_VARS = %w[HOME USER LANG].freeze
```

Comment:
Nit: personally I'd leave an empty line for visual separation
```suggestion
    WHITELISTED_VARS = %w[HOME USER LANG].freeze

```

Reply:
Ack, pushing the change.